### PR TITLE
Adding notebook tests

### PR DESCRIPTION
**Pull request by pgoiporia**

### CODE REVIEW

**Comment by singular-value** [comment]
(notebook test is failing)

File: examples/aqt_compile.ipynb [hunk 1]
```diff
@@ -26,7 +26,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "token = os.getenv(\"TEST_USER_TOKEN\")\n",
+    "# service = css.Service(api_key=\"YOUR API KEY\")\n",
+    "token = os.getenv(\"SUPERSTAQ_API_KEY\")\n",
```
Comment (vtomole) [inline examples/aqt_compile.ipynb:30]: You can remove this line and just call `service = css.Service()` on the following line.

**Review by vtomole — CHANGES_REQUESTED** [review]

File: examples/aqt_compile.ipynb (hunk 1 above)
Comment (pgoiporia) [reply examples/aqt_compile.ipynb:30]: By line you mean line 30?

Comment (vtomole) [reply examples/aqt_compile.ipynb:30]: Yeah, i think removing like 30 and calling `service = css.Service()` will pass notebooktests

**Comment by vtomole** [comment]
@pgoiporia Are we ready to update these notebook tests to use the code from `app_ss` now?

**Comment by pgoiporia** [comment]
@vtomole I can't tell my pytest is failing. It passes locally

File: check/pytest_.py [hunk 2]
```diff
@@ -4,5 +4,44 @@
...
+        os.environ["GOOGLE_CLIENT_ID"] = secret["google_client_id"]
+        os.environ["GOOGLE_CLIENT_SECRET"] = secret["google_client_secret"]
+        os.environ["PRANAV_IBMQ_TOKEN"] = secret["pranav_ibmq_token"]
+        os.environ["GOOGLE_API_KEY"] = secret["google_api_key"]
+    else:
+        raise ValueError(
+            "We currently only handle the secret value response having the SecretString key, "
+            "but this response doesn't have that key"
+        )
+
```
Comment (vtomole) [inline check/pytest_.py:33]: Delete this.

**Comment by vtomole** [comment]
Resolve merge conflicts as well

**Comment by stephanielee9** [comment]
Not relevant anymore

**Pull request closed without merging**